test/forms/ConsignmentReferencesSpec.scala

```diff
@@ -75,11 +76,11 @@ class ConsignmentReferencesSpec extends BaseSpec {
       val errors3 = ConsignmentReferences.form().fillAndValidate(inputData3).errors
       errors3 mustBe empty
```

Comment:
I would change those 2 lines into one. And if I remember correctly, we mostly use `must be()` instead of `mustBe`.